docs/DeveloperGuide.adoc

```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+|user
+|change my daily calorie budget
+|better reflect my new lifestyle requirements.
+
+|&#11088;&#11088;
+|user
+|modify my previous meal entries
+|fix missing or inaccurate information.
+
+|&#11088;&#11088;
+|user
+|delete my previous meal entries
+|undo mistakenly added meal entries.
+
+|&#11088;&#11088;
+|user
+|generate a summary of meals over the past month
+|review my overall progress in keeping to my meal plan.
+
+|&#11088;&#11088;
+|user
+|view meal suggestions that fit in my remaining budget.
+|figure out what I can eat while keeping to my plan.
+
+|&#11088;
+|user
+|create a new meal entry based of a meal I commonly eat
+|keep track of a meal I eat often.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
-
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
-
-[discrete]
-=== Use case: Delete person
+// [discrete]
+=== Use Case: Add meal entry
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-+
+1.	User enters meal information.
+2.	System adds entry into database.
+3.	System shows updated calorie budget for the day.
 Use case ends.
 
 *Extensions*
 
-[none]
-* 2a. The list is empty.
-+
-Use case ends.
-
-* 3a. The given index is invalid.
+* 1a.	The given meal entry has insufficient information (calories, meal name, etc.)
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 1a1.	CaloFit shows an error message.
 +
-Use case resumes at step 2.
+Use case restarts at step 1.
 
 _{More to be added}_
 
 [appendix]
-== Non Functional Requirements
-
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-_{More to be added}_
+== Non-functional Requirements
+.	Should work on any mainstream OS as long as it has Java 11 or above installed.
+.	Should be able to hold up to 1000 meal entries without a noticeable sluggishness in performance for typical usage.
+.	A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```

Comment:
Maybe this can be more concise, perhaps change to:  Users with higher typing speed can use the interface more efficiently ?